### PR TITLE
Apply patches to import refactor

### LINKED ISSUE #742: [staging] sites and areas of operation map not displaying where geom is OSM type "way"
[links #704]

`~/en/organization/view/952f9889-b03c-4eb3-8173-2a4f5fb50fa5/`

image

The unusual thing about this record is that the location used for the site is a `way` rather than a `relation` or a `node`. A `way` can resolve to a closed polygon or a line, so we have classed it geotype as `line` in our location model. The specific location in the import data is: `Trans-Ekulu (osm, line) 3111303b-1a02-4d5e-b190-33d0e2cdfa63`. It's actually the only `line` in this particular import dataset; others have more.

### PR DESCRIPTION
**Pull request by hancush**
## Overview

This PR:

- Fixes the SQL to re-enable optionally flushing the database on new import
- Updates deployment scripts to install necessary server-level packages
- Simplifies the sheets loop in the import management command
- Updates the unit detail page to support areas and sites being any type of feature (previously, it assumed sites were polygons and sites were points)

Closes #742

## Testing Instructions

These changes have already been tested locally and deployed to staging for client review. Only a code review is necessary.

### CODE REVIEW

File: organization/views.py [hunk 1]
```diff
@@ -109,21 +112,33 @@ def get_context_data(self, **kwargs):
...
-        context['associations'] = (ass.object_ref for ass in associations)
-        for association in associations:
-            geom = association.object_ref.area.get_value().value.geometry
-            area_obj = {
-                'geom': geom,
-                'name': association.object_ref.area.get_value().value.name
-            }
-            context['areas'].append(area_obj)
+        emplacements = context['organization'].emplacements
+        context['emplacements'] = list(em.object_ref for em in emplacements)
```
Comment (fgregg) [inline organization/views.py:116]: pretty minor but more idiomatic to do

```
[em.object_ref for em in emplacements]
```

File: organization/views.py [hunk 2]
```diff
@@ -109,21 +112,33 @@ def get_context_data(self, **kwargs):
...
+        ]
+
+        context['sites'] = serialize(
+            'geojson',
+            Location.objects.filter(id__in=site_ids),
+            geometry_field='geometry'
+        )
+
+        associations = context['organization'].associations
+        context['associations'] = list(ass.object_ref for ass in associations)
```
Comment (fgregg) [inline organization/views.py:130]: ditto

**Review by fgregg — APPROVED** [review]
there's a couple very small style things around list comprehensions. they are optional imo

**Comment by hancush** [comment]
Word! I'm all for imposing idioms anywhere we can in this code base, @fgregg, thanks very much for pointing them out. 🙃

**Pull request merged**